### ISSUE TITLE
testing of thread

### ISSUE THREAD

**Issue by levBagryansky** [objectionary/eo-threads#9]
To be sure `thread` dataizes it's argument `slow` after start we can implement `time` and `sleep` objects that helps to regulate dataization time `sleep` does it) and measure (`time` does it) :
`[millis] > sleep` stops dataization for `millis` milliseconds
`[] > time` is an analog of System.currentTimeMillis() in java.
So we can write
```
[] > test-it-dataizes-after-start
  thread > a
    sleep 3000
  thread > b
    sleep 3000
  memory 0 > m1
  memory 0 > m2
  seq > @
    m1.write
      time
    a.start
    b.start
    a.join
    b.join
    m2.write
      time
    gt.
      4000
      minus.
        m1
        m2
```
Here we can be sure `a` and `b` works in parallel.
@yegor256 @mximp What do you think?

**Comment by mximp** [comment]
@levBagryansky 
Maybe it's better to have `sleep` method as `thread` attribute. In this case we can use it with passing thread into executable (as suggested earlier by @yegor256):
```
[] > sleep-test
  thread > a
    [t]
      (t.sleep 3000) > @
```
For the `time` object - we've just implemented such function in `eo.sys`: objectionary/eo-sys#5

**Comment by levBagryansky** [comment]
@mximp
We can  address to thread by `^`.
Since `sleep` we be dataized in current thread it's not necessary make it `thread`'s attribute. But `thread.sleep` will give us ability to addres to foreign `thread`:
```
thread > a
  ...
thread > b
  seq > @
    a.sleep
    ...
```
I don't know is it ok for us or not

**Comment by mximp** [comment]
@levBagryansky `sleep` is an internal will of the execution thread. No one outside can _ask a thread to sleep_. A thread can be asked for interruption though (`interrupt` in Java). 
So maybe you're right: we can have object `org.eolang.threads.sleep` which dataization would just do nothing for provided number of millis.

**Comment by levBagryansky** [comment]
@yegor256 what do you thing?

**Comment by yegor256** [comment]
@levBagryansky I think it's a good idea to have `QQ.threads.sleep` object, which makes dataization pause for the given amount of _milliseconds_

**Comment by mximp** [comment]
@levBagryansky please create separate issue for implementing `sleep` object.

**Comment by levBagryansky** [comment]
@mximp ok
